packages/agent-toolkit/src/core/tools/platform-api-tools/workforms-tools/create-form-tool.ts

```diff
@@ -0,0 +1,64 @@
+import { z } from 'zod';
+import {
+  BoardKind,
+  CreateFormMutation,
+  CreateFormMutationVariables,
+} from '../../../../monday-graphql/generated/graphql';
+import { createForm } from '../../../../monday-graphql/queries.graphql';
+import { ToolInputType, ToolOutputType, ToolType } from '../../../tool';
+import { BaseMondayApiTool, createMondayApiAnnotations } from '../base-monday-api-tool';
+import { GraphQLDescriptions } from './workforms.consts';
+
+export const createFormToolSchema = {
+  destination_workspace_id: z.number().describe(GraphQLDescriptions.form.args.destinationWorkspaceId),
+  destination_folder_id: z.number().optional().describe(GraphQLDescriptions.form.args.destinationFolderId),
+  destination_folder_name: z.string().optional().describe(GraphQLDescriptions.form.args.destinationFolderName),
+  board_kind: z.nativeEnum(BoardKind).optional().describe(GraphQLDescriptions.form.args.boardKind),
+  destination_name: z.string().optional().describe(GraphQLDescriptions.form.args.destinationName),
+  board_owner_ids: z.array(z.number()).optional().describe(GraphQLDescriptions.form.args.boardOwnerIds),
+  board_owner_team_ids: z.array(z.number()).optional().describe(GraphQLDescriptions.form.args.boardOwnerTeamIds),
+  board_subscriber_ids: z.array(z.number()).optional().describe(GraphQLDescriptions.form.args.boardSubscriberIds),
+  board_subscriber_teams_ids: z
+    .array(z.number())
+    .optional()
+    .describe(GraphQLDescriptions.form.args.boardSubscriberTeamsIds),
+};
+
+export class CreateFormTool extends BaseMondayApiTool<typeof createFormToolSchema, never> {
```

Comment:
Please extend the README.md and packages/agent-toolkit/README.md files with information about newly added tools